requirements/test.in

```diff
@@ -2,15 +2,14 @@
 
 -r base.in                          # Core dependencies of edx-analytics-data-api
 
-coverage==4.4.1
-ddt==1.1.1
-diff-cover >= 0.9.9
+coverage
+ddt
+diff-cover
 django-dynamic-fixture
-mock==2.0.0
-pep257==0.7.0
+pep257
```

Comment:
Newer versions of this are being released as the `pydocstyle` package, if you want to try upgrading it even further.

Reply:
Done